diffkemp/simpll/DifferentialFunctionComparator.cpp

```diff
@@ -21,67 +22,100 @@
 /// Compare GEPs. This code is copied from FunctionComparator::cmpGEPs since it
 /// was not possible to simply call the original function.
 /// Handles offset between matching GEP indices in the compared modules.
-/// Assumes that the offset alignment is stored inside the GEP instruction as
-/// metadata.
+/// Uses data saved in ElementIndexToNameMap.
 int DifferentialFunctionComparator::cmpGEPs(
         const GEPOperator *GEPL,
         const GEPOperator *GEPR) const {
+    int OldFunctionResult = FunctionComparator::cmpGEPs(GEPL, GEPR);
```

Comment:
Maybe a better name would be OriginalResult? OldFunction seems like the function is deprecated.